integration-tests/tests/utils/docker_utils.py

```diff
@@ -0,0 +1,35 @@
+"""Provide utility functions related to the use of Docker during integration tests."""
+
+import subprocess
+
+from tests.utils.utils import get_binary_path
+
+
+def list_running_services_in_compose_project(project_name: str) -> list[str]:
+    """
+    Lists running Docker services that belong to the given Docker Compose project.
+
+    :param project_name:
+    :return: List of the running services that belong to the specified Docker Compose project.
+    """
+    docker_bin = get_binary_path("docker")
+
+    # fmt: off
+    compose_ps_cmd = [
+        docker_bin,
```

Comment:
can we simply write
```suggestion
    # fmt: off
    compose_ps_cmd = [
        "/usr/bin/env",
        "docker"
```
if the docker_bin absolute path isn't consumed (e.g. logged) anywhere?

Reply:
it doesn't really seem like there's a clear benefit to this change; plus, using `get_binary_path` gives a clear error message when Docker is not on PATH, so eliminating it would remove that. @junhaoliao could you elaborate?

Reply:
resolved offline